packages/grafana-data/src/types/panel.ts

```diff
@@ -203,6 +203,8 @@ export interface VisualizationSuggestion<TOptions = any, TFieldConfig = any> {
   transformations?: DataTransformerConfig[];
   /** Tweak for small preview */
   previewModifier?: (suggestion: VisualizationSuggestion) => void;
+  /** A value between 0-100 how suitable suggestion is */
```

Comment:
We should add some guidelines for what the absolute numbers mean

IE:
* 100 = total confidence this is the only panel that should render this data
* >80 = the data fits perfectly for the visualization
* >60 = data works well with this visualization
* >40 = with some transformations, it can work
* <40 = ? may not even be reasonable?

Reply:
@ryantxu 
yea, maybe an enum is better? 
* Best
* Good
* OK